cmd/openshift-install/create.go

```diff
@@ -96,8 +96,9 @@ var (
 
 				err = waitForBootstrapComplete(ctx, config, rootOpts.dir)
 				if err != nil {
+					logrus.Info("Installation failure detected. Running gather bootstrap...")
```

Comment:
Maybe `logrus.Error(err)` here instead?  Overlaps with the later fatal message, but makes it harder to skim past the important bit ;).